src/components/EventList.js

```diff
@@ -35,14 +36,19 @@ export const EventList = ({
     }
   };
 
-  const renderItem = useRenderItem(QUERY_TYPES.EVENT_RECORDS, navigation, { noSubtitle });
+  const renderItem = useRenderItem(QUERY_TYPES.EVENT_RECORDS, navigation, {
+    noSubtitle
+  });
```

Comment:
this was linted by you because of some eslint problems occurred within your development environment?

Reply:
I only got ESlint errors over punctuation marks.

Reply:
reverted this style right now together with you in pair review